src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -339,7 +339,13 @@
             {
                 "language": "aspnetcorerazor",
                 "scopeName": "text.aspnetcorerazor",
-                "path": "./syntaxes/aspnetcorerazor.tmLanguage.json"
+                "path": "./syntaxes/aspnetcorerazor.tmLanguage.json",
+                "embeddedLanguages": {
```

Comment:
What were the consequences of these not being explicit embeddedLanguages?

Reply:
What do you mean by "explicit"?

Reply:
Just that they weren't listed here.

Reply:
Ah as in prior to this. It meant that you didn't get bracket matching, indentation rules, contextual comments etc. in the embedded portions of the Razor document (definitely less than ideal 😄)